art/attacks/evasion/pixel_threshold.py

```diff
@@ -448,9 +480,7 @@ def _perturb_image(self, x: np.ndarray, img: np.ndarray) -> np.ndarray:
 To speed up predictions, the entire parameters array is passed to `self.func`,
 where a neural network model can batch its computations and execute in parallel
 Search for `CHANGES` to find all code changes.
-
 Dan Kondratyuk 2018
-
```

Comment:
Please don't remove this line, it's needed for correct formatting of the docs.

Reply:
Not required in the documentation.